addons/rose/app/styles/rose/components/filter-tags/_index.scss

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.filters-applied {
+  display: flex;
+  align-items: center;
+  margin-bottom: 1rem;
+
+  > * + * {
```

Comment:
What is the intent of this? I _think_ I get it (you're adding margin to the tags except the first since it won't have a sibling?) but this is quite a confusing nested selector and should be refactored

Reply:
This has changed after lovely feedback from a team member.